fw/rbcx-coprocessor/include/Bsp.hpp

```diff
@@ -36,33 +36,69 @@ inline const PinDef usbDpPullUpPin
 inline const PinDef controlUartTxPin = std::make_pair(GPIOC, GPIO_PIN_10);
 inline const PinDef controlUartRxPin = std::make_pair(GPIOC, GPIO_PIN_11);
-// inline const PinDef   tunnelUartTxPin = std::make_pair(GPIOC, GPIO_PIN_12);
-// inline const PinDef   tunnelUartRxPin = std::make_pair(GPIOD, GPIO_PIN_2);
-inline const PinDef tunnelUartTxPin = std::make_pair(GPIOA, GPIO_PIN_9);
-inline const PinDef tunnelUartRxPin = std::make_pair(GPIOA, GPIO_PIN_10);
 
-inline USART_TypeDef* const debugUart = USART1;
-inline USART_TypeDef* const userUart = USART2;
+inline const PinDef uart5TxPin = std::make_pair(GPIOC, GPIO_PIN_12);
+inline const PinDef uart5RxPin = std::make_pair(GPIOD, GPIO_PIN_2);
+
+inline USART_TypeDef* const tunnelUart = USART1;
+inline USART_TypeDef* const debugUart = USART2;
 inline USART_TypeDef* const servoUart = USART3;
 inline USART_TypeDef* const controlUart = UART4;
-inline USART_TypeDef* const tunnelUart = USART1; //UART5;
 
 inline DMA_Channel_TypeDef* const tunnelUartTxDmaChannel = DMA1_Channel4;
 inline DMA_Channel_TypeDef* const tunnelUartRxDmaChannel = DMA1_Channel5;
 
+#define DEBUGUART_TX_DMA_HANDLER DMA1_Channel7_IRQHandler
+inline const IRQn_Type debugUartTxDmaIRQn = DMA1_Channel7_IRQn;
+inline const unsigned debugUartTxDmaIrqPrio = 4;
+inline DMA_Channel_TypeDef* const debugUartTxDmaChannel = DMA1_Channel7;
+inline DMA_Channel_TypeDef* const debugUartRxDmaChannel = DMA1_Channel6;
+
 #define CONTROLUART_TX_DMA_HANDLER DMA2_Channel4_5_IRQHandler
 inline const IRQn_Type controlUartTxDmaIRQn = DMA2_Channel4_5_IRQn;
 inline const unsigned controlUartTxDmaIrqPrio = 4;
 inline DMA_Channel_TypeDef* const controlUartTxDmaChannel = DMA2_Channel5;
 inline DMA_Channel_TypeDef* const controlUartRxDmaChannel = DMA2_Channel3;
 
+#elif RBCX_HW_VERSION_11
+// TODO: add pins when HW finalized
+
+inline USART_TypeDef* const userUart = USART1;
+inline USART_TypeDef* const tunnelUart = USART2;
+inline USART_TypeDef* const controlUart = UART3;
+inline USART_TypeDef* const debugUart = USART4;
+inline USART_TypeDef* const servoUart = USART5;
+
+inline DMA_Channel_TypeDef* const tunnelUartTxDmaChannel = DMA1_Channel7;
+inline DMA_Channel_TypeDef* const tunnelUartRxDmaChannel = DMA1_Channel6;
+
+#define CONTROLUART_TX_DMA_HANDLER DMA1_Channel2_IRQHandler
+inline const IRQn_Type controlUartTxDmaIRQn = DMA1_Channel2_IRQn;
+inline const unsigned controlUartTxDmaIrqPrio = 4;
+inline DMA_Channel_TypeDef* const controlUartTxDmaChannel = DMA1_Channel2;
+inline DMA_Channel_TypeDef* const controlUartRxDmaChannel = DMA1_Channel3;
+
+#define DEBUGUART_TX_DMA_HANDLER DMA2_Channel4_5_IRQHandler
+inline const IRQn_Type debugUartTxDmaIRQn = DMA2_Channel4_5_IRQn;
+inline const unsigned debugUartTxDmaIrqPrio = 4;
+inline DMA_Channel_TypeDef* const debugUartTxDmaChannel = DMA2_Channel5;
+inline DMA_Channel_TypeDef* const debugUartRxDmaChannel = DMA2_Channel3;
+
+#else
+#error "No UART config for this HW"
+#endif
+
```

Comment:
Fakt tam chceš udržovat kompatibilitu takto? Bych to prostě přepsal, až budou nový desky.

Reply:
Jak říkám, je to jen nástřel, zatím je nemáme, a už jsem měl kus definic, který je na v1.1, tak jsem to tam dal takhle. Klidně se můžeme rozhodnout, že už v1.0 neexistuje, akorát tím ale vyřadíme @JarekParal :/

Reply:
Jen upozorňuji, že definice pro v1.1 budou finální, až bude deska ve výrobě ;-)

Reply:
Jarek pak zas dostane novou desku :) Nechal bych tam jen aktuální stav

Reply:
Odstraněno.